.github/workflows/main.yml

```diff
@@ -98,6 +98,19 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+
+    - name: Cache datasets
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-datasets
+        with:
+          path: |
+            data
+            ~/tensorflow_datasets
+          key: build-${{ env.cache-name }}-${{ github.ref }}
+          restore-keys: |
+            build-${{ env.cache-name }}-
+
```

Comment:
You have an identation issue in the yaml file.

Reply:
yup corrected